### ISSUE TITLE
Can't load the gophish admin page

### ISSUE THREAD

**Issue by hyperacti** [gophish/gophish#1407]
**Thanks for reaching out! We're happy to help resolve issues as quickly as possible.**

**Please use this template when creating a new issue. If you do not follow this template format, your issue may be closed without comment.**

Before filing a new issue, please use the search bar at the top of the browser to search for similar issues. Also, please make sure you have read our [documentation](https://getgophish.com/documentation/), which covers how to use Gophish.

What version of Gophish are you using?: 
0.7.1 64bit

Brief description of the issue: 
I dont even know. I'm new to all these. But check the errors

What are you expecting to see happen? :
Load the admin page. Work as expected

What are you seeing happen?
Error loading the page

If this question is related to email templates or landing pages not working as expected, please provide your template or landing page below:

```html
Insert code here
```

Please provide any terminal output that may be relevant below:
time="2019-03-23T18:05:10Z" level=info msg="Background Worker Started Successfully - Waiting for Campaigns"
time="2019-03-23T18:05:10Z" level=warning msg="No contact address has been configured."
time="2019-03-23T18:05:10Z" level=warning msg="Please consider adding a contact_address entry in your config.json"
goose: no migrations to run. current version: 20180830215615
time="2019-03-23T18:05:10Z" level=info msg="Starting phishing server at http://0.0.0.0:80"
time="2019-03-23T18:05:10Z" level=fatal msg="listen tcp 0.0.0.0:80: bind: address already in use"

```
Insert output here
```

Please provide as many steps as you can to reproduce the problem:
 * Step 1 - ./gophish
 * Step 2 -

**Comment by jordan-wright** [comment]
Hi there!

You'll notice the message "listen tcp 0.0.0.0:80: bind: address already in use". This means that there is already a webserver listening on TCP port 80 (such as Apache or nginx). You'll need to shut that down before starting Gophish.

You can find other similar issues [here](https://github.com/gophish/gophish/search?q=%22address+already+in+use%22&unscoped_q=%22address+already+in+use%22&type=Issues).

Hope this helps!
